src/event.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * simple library for describing events and waiting for events
```

Comment:
This seems to currently be used primarily for game start and game stop. Is there plans to do more this this?

Reply:
You're correct that it's currently just for game start/stop, but it's super general. We could refactor some of the request-response networking code to use `event.h`. I am hoping it will be useful in the future as well, but I don't have any other specific ideas.

Reply:
I see, sounds good to me! I bet we will find more uses for this.